src/Model/Indexing/IndexNames.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Model\Indexing;
+
+enum IndexNames: string
+{
+    case Events = 'events';
+    case Organization = 'organization';
+
+    public static function values(): array
+    {
+        return array_column(IndexNames::cases(), 'value');
```

Comment:
This is clever! And I didn't know (until now) that you can use `array_column` on objects. Cool!